.gitignore

```diff
@@ -9,6 +9,7 @@ dist/
 package-lock.json
```

Comment:
The sha in this commit message (Copy patternfly import files from cockpit project automatically ) is incorrect. The correct one is https://github.com/cockpit-project/starter-kit/commit/203604a463f900cfef0980a629ddfe9fb14c39ad

Reply:
Ugh, thanks for spotting! Fixed, can you please re-approve?